docs/releaseNotes.yml

```diff
@@ -74,14 +74,18 @@ items:
           deny traffic when it is unable to communicate to the RateLimitService 
           returning a 500.
         docs: https://www.getambassador.io/docs/emissary/latest/topics/running/services/rate-limit-service/
-
       - title: Properly populate alt_state_name for Tracing, Auth and RateLimit Services
         type: bugfix
         body: >-
           Previously, setting the <code>stats_name</code> for the <code>TracingService</code>, <code>RateLimitService</code> 
           or the <code>AuthService</code> would have no affect because it was not being properly passed to the Envoy cluster
           config. This has been fixed and the <code>alt_stats_name</code> field in the cluster config is now set correctly.
           (Thanks to <a href="https://github.com/psalaberria002">Paul</a>!)
+      - title: Add support for config change batch window before reconfiguring Envoy
+        type: feature
+        body: >-
+          The AMBASSADOR_RECONFIG_MAX_DELAY env var can be optionally set to batch changes for the specified 
```

Comment:
I think we might want to wrap AMBASSADOR_RECONFIG_MAX_DELAY in `<code>AMBASSADOR_RECONFIG_MAX_DELAY</code>` just to keep the changelog consistent with how the other env vars are displayed

Reply:
Let's make sure we get this in the Docs too. In the release notes and Alice's new Environment page